src/pages/greeting/Greeting.js

```diff
@@ -0,0 +1,31 @@
+import './greeting.scss';
+
+import { Infobar } from '../../components/infoBar';
+
+export const Greeting = ({ user }) => {
+  const time = new Date().getHours();
+  let dayTime;
+
+  if (time >= 3 && time <= 12) dayTime = 'morning';
+  else if (time > 12 && time < 18) dayTime = 'afternoon';
+  else if (time >= 18 && time <= 22) dayTime = 'evening';
+  else dayTime = 'night';
```

Comment:
do you need that code?

Reply:
yes, to detect the time of day

Reply:
or, maybe, is here something wrong?

Reply:
I mean I don't remember that component in the project design